atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -10,21 +10,23 @@
 import time
 import platform
 import random
+from typing import Dict
 
-from atomic_reactor.constants import (IMAGE_TYPE_DOCKER_ARCHIVE, IMAGE_TYPE_OCI, IMAGE_TYPE_OCI_TAR,
+from atomic_reactor.constants import (IMAGE_TYPE_DOCKER_ARCHIVE, IMAGE_TYPE_OCI,
                                       DOCKER_PUSH_MAX_RETRIES, DOCKER_PUSH_BACKOFF_FACTOR)
 from atomic_reactor.config import get_koji_session
 from atomic_reactor.plugin import PostBuildPlugin
 from atomic_reactor.plugins.pre_fetch_sources import PLUGIN_FETCH_SOURCES_KEY
 from atomic_reactor.metadata import annotation_map
+from atomic_reactor.plugins.prepub_flatpak_create_oci import FlatpakCreateOciPlugin
```

Comment:
Is it it safe to import the flatpak plugin unconditionally? I don't know if the flatpak deps are treated as optional or not :thinking: 

As an alternative solution - plugins that make their results available to other plugins typically define the plugin key in `atomic_reactor.constants`. Instead of importing the whole plugin and using `FlatpakCreateOciPlugin.key`, you could just import the key from constants.